### PR TITLE
docs(manager/flux): improve readme

### PR DESCRIPTION
**Pull request by gberche-orange**
## Documentation 

Highlight current manager support with headers

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] render markdown to preview to https://docs.renovatebot.com/modules/manager/flux/

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: lib/modules/manager/flux/readme.md [hunk 1]
```diff
@@ -44,4 +54,6 @@ If instead you have all your Flux manifests inside a `flux/` directory, you woul
 }
 ```
 
+### Versionning
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:57]:
```suggestion
### Versioning
```

File: lib/modules/manager/flux/readme.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
-This manager parses [Flux](https://fluxcd.io/) YAML manifests and:
+This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:
+1. [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) resources
+2. Flux [system](https://fluxcd.io/docs/installation) manifests
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:3]:
```suggestion
This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:

1. [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) resources
1. Flux [system](https://fluxcd.io/docs/installation) manifests

```

This should fix the linting problem.
We can just rely on the site generator to number ordered lists for us. 😉

File: lib/modules/manager/flux/readme.md [hunk 3]
```diff
@@ -16,12 +19,19 @@ In addition, for the `flux` manager to properly link `HelmRelease` and `HelmRepo
 
 Namespaces will not be inferred from the context (e.g. from the parent `Kustomization`).
 
+### Flux system support
+
+Support updating flux system manifests generated during [flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:24]:
```suggestion
Support updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
```

File: lib/modules/manager/flux/readme.md [hunk 4]
```diff
@@ -16,12 +19,19 @@ In addition, for the `flux` manager to properly link `HelmRelease` and `HelmRepo
 
 Namespaces will not be inferred from the context (e.g. from the parent `Kustomization`).
 
+### Flux system support
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:22]:
```suggestion
### Flux system manifests support
```

File: lib/modules/manager/flux/readme.md [hunk 5]
```diff
@@ -16,12 +19,19 @@ In addition, for the `flux` manager to properly link `HelmRelease` and `HelmRepo
...
+
+Support updating flux system manifests generated during [flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+
 Updating system manifests requires that either:
 
 1. The `flux` tool is pre-installed, or
 2. You run a Docker image based on [containerbase](https://github.com/containerbase), such as the official Renovate images, and have `binarySource=install` configured
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:29]:
```suggestion
1. The `flux` tool is pre-installed, or
1. You run a Docker image based on [containerbase](https://github.com/containerbase), such as the official Renovate images, and have `binarySource=install` configured
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: lib/modules/manager/flux/readme.md [hunk 6]
```diff
@@ -1,7 +1,12 @@
-This manager parses [Flux](https://fluxcd.io/) YAML manifests and:
+This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:
 
-1. Extracts `helm` dependencies from `HelmRelease` resources
-2. Extracts `github-releases` dependencies from system manifests (`flux-system/gotk-components.yaml` files) and regenerates them when new versions of Flux are available
+1. [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) resources
+1. Flux [system](https://fluxcd.io/docs/installation) manifests
+
```
Comment (HonkingGoose) [inline lib/modules/manager/flux/readme.md:5]:
```suggestion
```

**Comment by gberche-orange** [comment]
thanks @HonkingGoose !

**Comment by HonkingGoose** [comment]
Cheers @gberche-orange! 😄

**Review by rarkins — APPROVED** [review]

**Pull request merged**